charts/jindofs/README.md

```diff
@@ -492,7 +492,7 @@ tar -zxvf b2sdk-3.3.2.tar.gz
 
 ```
 $ cd /root
-$ wget https://mirror.bit.edu.cn/apache/spark/spark-2.4.6/spark-2.4.6-bin-hadoop2.7.tgz
+$ wget https://mirrors.bit.edu.cn/apache/spark/spark-2.4.6/spark-2.4.6-bin-hadoop2.7.tgz
```

Comment:
now can`t find spark-2.4.6, how about change it to `https://mirrors.bit.edu.cn/apache/spark/`

Reply:
Thanks. Updated.